sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -102,12 +184,28 @@ export interface Entity {
     text: string;
 }
 
+// @public (undocumented)
+export interface EntityLinkingResult {
+    documents: DocumentLinkedEntities[];
+    errors: DocumentError[];
+    modelVersion: string;
+    statistics?: TextDocumentBatchStatistics;
+}
+
 // @public
 export type ErrorCode = ErrorCodeValue | InnerErrorCodeValue;
 
 // @public
 export type ErrorCodeValue = "InvalidRequest" | "InvalidArgument" | "InternalServerError" | "ServiceUnavailable";
 
+// @public
+export type ExtractKeyPhraseseResultResponse = ExtractKeyPhrasesResultArray & {
+    _response: coreHttp.HttpResponse & {
```

Comment:
I understand why this might be useful, but most of our clients are not providing this information back to our customers. Is this how our clients are expected to respond? If so, I believe we should make some issues to update the clients that aren't doing this today.

cc: @bterlson , @ramya-rao-a

Reply:
I like the idea of adding `coreHttp.HttpResponse` so folks can see headers and status code, but I'm less convinced we need to add things that come from `HttpOperationResponse` (such as parsedBody) since that's lower-level transport stuff that we might want to change internally to make serialization better.

I talked to @deyaaeldeen a bit and I think we should discuss this as a team tomorrow.

Reply:
@sadasant I agree that we should be consistent. Some packages provide them and some do not.

Reply:
Thank you for your responses! I'm excited for tomorrow's meeting 🌞